src/main/java/InvalidDeadlineException.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * Identified the exception of invalid deadline input
+ */
+public class InvalidDeadlineException extends Exception{
```

Comment:
This class does not seem to do anything different from its parent. Consider removing redundant classes or updating it accordingly.